src/languages/nix.js

```diff
@@ -45,6 +45,10 @@ export default function(hljs) {
     end: /\}/,
     keywords: KEYWORDS
   };
+  const ESCAPED_DOLLAR = {
+    className: 'char.escape',
+    begin: /''\$\{/,
```

Comment:
Is `''` escaping the `$` or `${`... your issue seems to indicate it's escaping `$`, so I'm a little confused at seeing the { here.

Reply:
I think I've confused myself here a little. The only reason you would ever want to escape the dollar, is to escape an antiquote. But it is indeed the dollar itself that should be escaped. Updating now.